### PR TITLE
FOUR-14077: Update the URL of Launchpad

### PR DESCRIPTION
**Pull request by pmPaulis**
## Issue & Reproduction Steps
Update the URL of Launchpad

## Solution
- List the changes you've introduced to solve the issue.

## How to Test
Review the URL for processes

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-14077

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fd82a7058d.engk8s.processmaker.net

**Comment by pmPaulis** [comment]
@eiresendez 
Please Could you doble check this change?
Regards,

**Review by eiresendez — APPROVED** [review]

**Comment by ryancooley** [comment]
@pmPaulis Could this URL be hardcoded anywhere that might cause an instance on 4.9.x to fail when upgraded to 4.9.2?

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fd82a7058d.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-fd82a7058d.engk8s.processmaker.net

File: routes/web.php [hunk 1]
```diff
@@ -93,8 +93,15 @@
 
     Route::get('designer/scripts/categories', [ScriptController::class, 'index'])->name('script-categories.index')->middleware('can:view-script-categories');
     Route::get('designer', [DesignerController::class, 'index'])->name('designer.index');
-
-    Route::get('processes-catalogue/{process?}', [ProcessesCatalogueController::class, 'index'])->name('processes.catalogue.index')->middleware('can:view-process-catalog');
+    Route::get('processes-catalogue/{process?}', [ProcessesCatalogueController::class, 'redirect'])
+        ->name('processes.catalogue.index')
+        ->middleware('can:view-process-catalog');
+    Route::get('process-catalogue/{process?}', [ProcessesCatalogueController::class, 'redirect'])
```
Comment (pmPaulis) [inline routes/web.php:99]: remove this Route::get('process-catalogue/{process?}', [ProcessesCatalogueController::class, 'redirect'])
        ->name('process.catalogue.index')
        ->middleware('can:view-process-catalog');

File: routes/web.php [hunk 2]
```diff
@@ -94,7 +94,16 @@
...
-    Route::get('processes-catalogue/{process?}', [ProcessesCatalogueController::class, 'index'])->name('processes.catalogue.index')->middleware('can:view-process-catalog');
+     Route::get('process-browser/{process?}', [ProcessesCatalogueController::class, 'index'])
+        ->name('process.browser.index')
+        ->middleware('can:view-process-catalog');
+    //------------------------------------------------------------------------------------------
+    // Below route is for backward compatibility with old format routes. PLEASE DO NOT REMOVE
+    //------------------------------------------------------------------------------------------
+    Route::get('processes-catalogue/{process?}', function ($process = null) {
+        return redirect()->route('process.browser.index', [$process]);
+    })->name('processes.catalogue.index.redirect');
```
Comment (caleeli) [inline routes/web.php:105]: Consider to keep the same previous route name to keep the compatibility

**Review by caleeli — APPROVED** [review]

File: routes/web.php (hunk 2 above)
Comment (pmPaulis) [reply routes/web.php]: okas

**Comment by pmPaulis** [comment]
@ryancooley 
We have added defensive code for the use case you mentioned

**Pull request merged**